portfolio/src/main/webapp/index.html

```diff
@@ -151,11 +151,21 @@ <h2>Contact</h2>
           <a href="https://www.linkedin.com/in/cindyhao/"target=”_blank”  class="link">LinkedIn</a>
           <a href="https://www.cindyhao.com"target=”_blank”  class="link">Another website!</a>
         </div>
+
+        <!-- Form to Submit Comment -->
+        <div id="form-container" class="container">
+          <form id="comment-form" action="/data" method="POST">
+            <label for="form-comment-input">Comment</label>
+            <input type="text" name="form-comment" id="form-comment-input" onkeyup="configureBtn()">
```

Comment:
I don't know if keyup events work with virtual keyboard and mobile devices, but I think that is fine for now :)